src/test/scala/com/newlibertie/pollster/api/v1/PollApiSpec.scala

```diff
@@ -46,14 +45,13 @@ class PollApiSpec extends WordSpec with Matchers with ScalatestRouteTest with La
       """.stripMargin
 
       val poll = Poll(pollDefinitionStr)
-      val pollId: String = Poll.write(poll) match {
+      val pollId: String = poll.create() match {
         case None => ""//Or handle the lack of a value another way: throw an error, etc.
```

Comment:
please add comments to the main methods and main algorithm blocks indicating what is being done